### PR TITLE
created dummy register page

### LINKED ISSUE #10: Register Page (Dummy)
## Tasks
- [x] Create a Register.js component in client/routes
- [ ] Add a /register route to App.js with react-router
- [x] Create the Register page according to the screen in Figma (attach below)

## Acceptance Criteria
- Going to [url]/register routes to the Register component
- Register page should look like this (attach Figma screenshot):
 
## Resources
- http://www.hackingwithreact.com/read/1/22/how-to-add-a-new-route-to-react-router

### PR DESCRIPTION
**Pull request by DualSaturn**
## Description
Resolves #10 

## Testing
I have included screenshots of the page.
dummy-register-page-screenshot

### CODE REVIEW

**Review by vung1 — APPROVED** [review]

File: client/src/App.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.App {
+/* .App {
```
Comment (cqvu) [inline client/src/App.css:1]: If unused, can you delete?

File: client/src/components/Register.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react'
+import './Register.css'
+
+const Register = () => {
+    return (
+        <div className='Register'>
```
Comment (cqvu) [inline client/src/components/Register.js:6]: Nit: lowercase on 'Register' for consistency

File: client/src/index.css [hunk 3]
```diff
@@ -1,10 +1,11 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+  /* font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (cqvu) [inline client/src/index.css:3]: remove comment if unused

**Review by cqvu — COMMENTED** [review]
Some minor comments, otherwise looks good!

File: client/src/index.css (hunk 3 above)
Comment (DualSaturn) [reply client/src/index.css]: Done!

File: client/src/components/Register.js (hunk 2 above)
Comment (DualSaturn) [reply client/src/components/Register.js]: Changed it!

File: client/src/App.css (hunk 1 above)
Comment (DualSaturn) [reply client/src/App.css]: Took out the commented code

**Review by cqvu — APPROVED** [review]
Looks good! 🚀

**Review by vung1 — APPROVED** [review]

**Pull request merged**